app/styles/components/_global.scss

```diff
@@ -15,16 +15,24 @@ button {
   -webkit-font-smoothing: antialiased;
   background-color: $base-link-color;
   border-radius: $base-border-radius;
-  font-size: $base-font-size * 0.75;
+  font-size: $base-font-size * 0.85;
   color: white;
   border: none;
   cursor: pointer;
   display: inline-block;
-  padding: 0.75em 1em;
+  padding: 0.3em 1em;
   vertical-align: middle;
   white-space: nowrap;
 }
 
+h2 {
+  font-size: 20px;
```

Comment:
All font sizes should be in em.  Its better if they are in a percent of $base-font-size as on line 18, but em's for sure.